packages/block-editor/src/components/block-edit/index.js

```diff
@@ -1,69 +1,39 @@
-/**
- * External dependencies
- */
-import memize from 'memize';
-
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { useMemo } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Edit from './edit';
 import { BlockEditContextProvider, useBlockEditContext } from './context';
 
-class BlockEdit extends Component {
-	constructor() {
-		super( ...arguments );
-
-		// It is important to return the same object if props haven't changed
-		// to avoid  unnecessary rerenders.
-		// See https://reactjs.org/docs/context.html#caveats.
-		this.propsToContext = memize( this.propsToContext.bind( this ), {
-			maxSize: 1,
-		} );
-	}
-
-	propsToContext(
+function BlockEdit( props ) {
```

Comment:
We could use destructuring here in the function parameters, as we do for other React components.

Also, we can do `export default function ...` here.

Reply:
We need `props`.

Reply:
Ah, I see; I missed that.